charts/ocis/templates/appprovider/deployment.yaml

```
@@ -72,6 +72,10 @@ spec:
               value: {{ $.Values.features.appsIntegration.wopiIntegration.wopiServerURI | quote }}
             - name: APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL
               value: {{ tpl $.Values.features.appsIntegration.wopiIntegration.wopiFolderURI $ | quote }}
+            {{- if ne $.Values.features.appsIntegration.wopiIntegration.wopiFolderURIPathTemplate nil }}
```

Why not using with here?

```suggestion
            {{- with $.Values.features.appsIntegration.wopiIntegration.wopiFolderURIPathTemplate }}
```

---

I want to allow "" (empty string). That's not possible with `with`

---

Makes sense.